lxqt-base/lxqt-powermanagement/lxqt-powermanagement-0.16.0-r1.ebuild

```diff
@@ -8,6 +8,8 @@ inherit cmake
 DESCRIPTION="LXQt daemon for power management and auto-suspend"
 HOMEPAGE="https://lxqt.github.io/"
 
+MY_PV="$(ver_cut 1-2)*"
```

Comment:
hm, that's a bit oddly named when not used for the ebuild itself. how about `LXQTPV' instead or similar?

Reply:
Works for me.  I just copied this from one of the other ebuilds, for which I could change it probably the next time I make any changes to them.